### PR TITLE
Feature/full website

### PR DESCRIPTION
**Pull request by tnyandoro**
## Project requirements ##
In previous projects, i created the core functionality of the app: a calculator component. Now i have developed a full website with several pages.

- Created a website consisting of 3 pages: Home, Calculator, and Quote.
The general layout matches these wireframes:
Add your own styles to improve the look and feel.
In the lesson about styling, we introduced several ways to add styles in a React application:
React basic styling (inline styles, importing CSS and pre-processed CSS files).
CSS modules.
Styled components.
In this project, you should use React basic styling.
For every page, i created a route.
Make sure that navigation links for all 3 routes are displayed on each page.
Used React Router to make the routes work.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  margin: 1px;
+}
+
+.content {
+  border: 2px solid rgb(108, 108, 108);
+  background-color: #e0e0e0;
+  color: #858694;
+  margin: 1%;
+}
+
```
Comment (Emmyn5600) [inline src/App.css]: - [x] Let us improve as well as modify our styling to have great project, I suggest you to change the height of calculator as well as display as flex with the texts `Lets do some calculations` that are displayed on the left.

react15

- [x] Update the live link demo, seem to be broken.

react18

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED :recycle:

Hi, @errea 

Great work on implementing the project requirements, however there are some issues to be solved before you get approved but you are almost there:muscle:. 
   
   - The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
- Your project looks awesome:heavy_check_mark:

Changes required :writing_hand: 
 
      * #### Please check the comments under the review.
       
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: src/App.css (hunk 1 above)
Comment (tnyandoro) [reply src/App.css]: - The Calculator ppage has been fixed
- The live link been resolved

**Review by Emmyn5600 — APPROVED** [review]
Hi, 

Good job so far! 👏🏼 @tnyandoro 

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
- Live link demo is running properly:heavy_check_mark:
- Your project looks awesome:heavy_check_mark:

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous reviewer.
 
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: src/App.css (hunk 1 above)
Comment (Emmyn5600) [reply src/App.css]: Well solved

**Pull request merged**